cli/diffParser.js

```diff
@@ -72,11 +71,11 @@ function convertToDiffObject(granule) {
         //large edit, doesn't really matter, either way,
         //just trim the info accordingly
         return {changeType: 'new', objectID: granule.fileInfo.newID,
-                parent: granule.fileInfo.newParent, content: granule.versions.new};
+                parent: granule.fileInfo.newParent, content: "<ins>" + granule.versions.new + "</ins>"};
```

Comment:
This won't make the **ins** and **del** be on a per line/sentence basis, but on a per node basis

For where to add this to make it on a per change/line/sentence basis, add it at line 284 or so, the method *getFileVersion*